### DISCUSSION TITLE
CVE-2023-38545

### DISCUSSION THREAD

**Discussion by sbergandy** [quarkusio/quarkus#36475] [Community]
Is there any announcement or new quarkus-micro-image base image for the  CVE-2023-38545 issue?

**Comment by maxandersen** [comment]
Working on it.

**Reply by sbergandy** [reply]
Hi @maxandersen Thank you for a quick answer. I'm happy to hear that.

**Reply by sbergandy** [reply]
Hello @maxandersen again. I would like to ask you if you were able to find any time for this issue? Many thanks for any answer in advance.

**Comment by gsmet** [comment]
We rebuild the image regularly to get the security fixes so I would expect it to be fixed in the image already. @cescoffier might know more.

See https://quay.io/repository/quarkus/quarkus-micro-image?tab=tags&tag=1.0

**Reply by cescoffier** [reply]
The latest micro image was republished three days ago. The scanner does not detect any CVE in this version.
